### PR TITLE
Improve instructor guide

### PR DESCRIPTION
**Pull request by bast**
Work in progress. I am adjusting both the exercise and the instructor guides on the same branch/PR since they are coupled.

### CODE REVIEW

**Comment by bast** [comment]
Will create the new exercise in a separate PR. It's more decoupled than it seemed.

File: _guides/guide.md [hunk 1]
```diff
@@ -38,15 +47,10 @@ Then delete the copied exercise repositories from a previous workshop.
...
-- Your old repository’s clone URL: [https://github.com/coderefinery/template-centralized-workflow-exercise](https://github.com/coderefinery/template-centralized-workflow-exercise)
-- Your new repository details: [{{ site.centralized_workflow_exercise_url }}]({{ site.centralized_workflow_exercise_url }})
-
-Forking exercise:
-- Your old repository’s clone URL: [https://github.com/coderefinery/template-forking-workflow-exercise](https://github.com/coderefinery/template-forking-workflow-exercise)
-- Your new repository details: [{{ site.forking_workflow_exercise_url }}]({{ site.forking_workflow_exercise_url }})
+Now create the two exercises by [importing](https://help.github.com/en/articles/creating-a-repository-from-a-template)
+these two templates:
+- [https://github.com/coderefinery/template-centralized-workflow-exercise](https://github.com/coderefinery/template-centralized-workflow-exercise)
+- [https://github.com/coderefinery/template-forking-workflow-exercise](https://github.com/coderefinery/template-forking-workflow-exercise)
```
Comment (rkdarst) [inline _guides/guide.md]: If we mostly use this for CodeRefinery, should we make clear what the default CodeRefinery names are so it matches the default CodeRefinery lesson?

Comment (bast) [reply _guides/guide.md]: Oh yes. This somehow fell of the desk during my refactoring. I will add a sentence on that.

Comment (bast) [reply _guides/guide.md]: I have added a word but think that it is clear from the guide now which repositories to create. The names will automatically change according to the configuration in `_config.yml`.

**Comment by wikfeldt** [comment]
looks good

**Pull request merged**